src/python/pants/engine/internals/tls_testing/generate_certs.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Run this script in this dir to generate the certs and keys needed for this test.
+# Note that on MacOS you will need a recent-ish homebrewed openssl, not the system one.
+
+set -xeuo pipefail
```

Comment:
Is `-x` leftover?

Reply:
No, it felt more useful in this case to echo everything.